tests/components/FilterSearch.test.tsx

```diff
@@ -0,0 +1,256 @@
+    render (<FilterSearch searchFields={[searchFieldsProp]}/>);
+
+    const searchBarElement = screen.getByRole('textbox');
+    userEvent.type(searchBarElement, 'J');
+
+    const firstAutocompleteSuggestion = screen.getByText('John Doe');
+    expect(firstAutocompleteSuggestion).toBeDefined();
+
+    const secondAutocompleteSuggestion = screen.getByText('Jane Doe');
+    expect(secondAutocompleteSuggestion).toBeDefined();
+
+    userEvent.type(searchBarElement, '{arrowdown}');
+    expect(searchBarElement).toHaveValue('John Doe');
+
+    userEvent.type(searchBarElement, '{arrowdown}');
+    expect(searchBarElement).toHaveValue('Jane Doe');
+  });
+
+  it('calls executeSearch when an autocomplete suggestion is selected using the arrow keys and "Enter"', () => {
+    render (<FilterSearch searchFields={[searchFieldsProp]}/>);
+
+    const searchBarElement = screen.getByRole('textbox');
+    userEvent.type(searchBarElement, 'J');
+
+    const firstAutocompleteSuggestion = screen.getByText('John Doe');
+    expect(firstAutocompleteSuggestion).toBeDefined();
+
+    userEvent.type(searchBarElement, '{arrowdown}');
+    expect(searchBarElement).toHaveValue('John Doe');
+    userEvent.type(searchBarElement, '{enter}');
+    expect(setFilterOption).toBeCalled();
+    expect(setOffset).toBeCalled();
+    expect(searchOperations.executeSearch).toBeCalled();
+  });
+
+  it('does not trigger executeSearch when "Enter" is pressed if an autocomplete suggestion is not selected with the arrow keys first, even if autocomplete values are returned', () => {
+    render (<FilterSearch searchFields={[searchFieldsProp]}/>);
+
+    const searchBarElement = screen.getByRole('textbox');
+    userEvent.type(searchBarElement, 'J');
+
+    const firstAutocompleteSuggestion = screen.getByText('John Doe');
+    expect(firstAutocompleteSuggestion).toBeDefined();
+
+    const secondAutocompleteSuggestion = screen.getByText('Jane Doe');
+    expect(secondAutocompleteSuggestion).toBeDefined();
+
+    userEvent.type(searchBarElement, '{enter}');
+    expect(setFilterOption).not.toBeCalled();
+    expect(setOffset).not.toBeCalled();
+    expect(searchOperations.executeSearch).not.toBeCalled();
+  });
+
+  it('calls executeSearch when an autocomplete suggestion is clicked with a cursor', () => {
+    render (<FilterSearch searchFields={[searchFieldsProp]}/>);
+
+    const searchBarElement = screen.getByRole('textbox');
+    userEvent.type(searchBarElement, 'J');
+    const firstAutocompleteSuggestion = screen.getByText('John Doe');
+    expect(firstAutocompleteSuggestion).toBeDefined();
+
+    userEvent.click(firstAutocompleteSuggestion);
+    expect(setFilterOption).toBeCalled();
+    expect(setOffset).toBeCalled();
+    expect(searchOperations.executeSearch).toBeCalled();
+  });
+});
+
+describe('tests using data without section labels', () => {
+  it('populates the dropdown menu with autocomplete results without section labels if they are not provided', () => {
```

Comment:
nit: maybe just `populates the dropdown menu with autocomplete results` since describe() states the data is without section label? I'm also fine if we can combine this and the first `describe` section together